CMakeLists.txt

```diff
@@ -208,6 +208,13 @@ if(DOXYGEN_FOUND)
     )
 endif(DOXYGEN_FOUND)
 
+########################################################################
+# Detect /lib versus /lib64
+########################################################################
+if (CMAKE_INSTALL_LIBDIR MATCHES lib64)
+    set(LIB_SUFFIX 64)
```

Comment:
Do we provide an option for `LIB_SUFFIX` to be set elsewhere before this? If so, please check for whether the variable is set & if so then if it is -not- "64" then print a warning & then overwrite the variable as noted here.

Reply:
`[balister@thuvia build (fix-lib64)]$ grep LIB_SUFFIX ../CMakeLists.txt 
    set(LIB_SUFFIX 64)
set(libdir "\${exec_prefix}/lib${LIB_SUFFIX}")
    DESTINATION lib${LIB_SUFFIX}/pkgconfig
set(VOLK_LIBRARY_DIR   lib${LIB_SUFFIX})
    set(CMAKE_MODULES_DIR lib${LIB_SUFFIX}/cmake)
[balister@thuvia build (fix-lib64)]$`

`[balister@thuvia build (fix-lib64)]$ grep LIB_SUFFIX ../CMakeLists.txt 
    set(LIB_SUFFIX 64)
set(libdir "\${exec_prefix}/lib${LIB_SUFFIX}")
    DESTINATION lib${LIB_SUFFIX}/pkgconfig
set(VOLK_LIBRARY_DIR   lib${LIB_SUFFIX})
    set(CMAKE_MODULES_DIR lib${LIB_SUFFIX}/cmake)
[balister@thuvia build (fix-lib64)]$`

I can't find any signs of anyone else setting LIB_SUFFIX

Reply:
So the question is, how will this interact if you use this volk as a submodule for gnuradio? Will we bump the git has of volk on  gnuradio-3.8.2?

Reply:
Pretty sure we will never use this Volk as a submodule to GR. That would have to be GR 3.8, which is coming into bug-fix-mode only soon isn't it? I am so out of the GR dev loop at the moment; too saturated with work!

Reply:
I'd say let's leave the code as-is now. It's a good starting point & we can always fix it up down the road if for some reason GR does update the submodule to this commit / release.